frontend/src/views/CanteenEditor/CanteenForm.vue

```diff
@@ -584,19 +514,8 @@ export default {
           }
         })
         .catch((e) => {
-          if (e.jsonPromise) {
-            e.jsonPromise.then((json) => {
-              const isDuplicateSiret = json.detail === "La resource que vous souhaitez créer existe déjà"
-              if (isDuplicateSiret) {
-                this.duplicateSiretCanteen = json
-                this.messageTroubleshooting = `Je veux ajouter une deuxième cantine avec le même SIRET : ${payload.siret}...`
-              }
-            })
-            this.$store.dispatch("notifyRequiredFieldsError")
-          } else {
-            this.$store.dispatch("notifyServerError", e)
-          }
-          window.scrollTo(0, 0)
+          // TODO
+          console.log(e)
```

Comment:
Ajout d'un message d'erreur peut-être ?

Reply:
Just add a `this.$store.dispatch("notifyServerError", e)`

Reply:
Fix : https://github.com/betagouv/ma-cantine/pull/2202/commits/1a8189bb690e52a8bd8b042c8cd62c90f216216e